app/src/main/java/com/mobileproto/hireddit/hireddit/speech/WordToSpeech.java

```diff
@@ -41,6 +41,12 @@ public void speak(String toSpeak)
         }
     }
 
+    public void pause() {
+        if (mTtobj.isSpeaking()) {
+            mTtobj.stop();
+        }
+    }
+
     public void stop()
```

Comment:
That's true, there's no way to resume. Sorry for the confusion. pause() stops the speech while stop() destroy the object.